cypress/integration/assign-automatic-tag.spec.js

```diff
@@ -0,0 +1,45 @@
+import * as data from '../../cypress.json';
+describe('Assign Automatic tags', function() {
+    beforeEach(function() {
+        cy.login(data.email, data.password);
+        // cy.visit(data.baseUrl + "/core/setting/tagsetting");
+    });
+    afterEach(function() {
+        cy.logout();
+    });
+    //it should visit setting/imap seting page
+    it('Add imap email and change their status', function () {
+        cy.addImap();
+        cy.get("#switchState").click();
+    })
+    // add a job probile
+    it('Add a job profile', function () {
+        cy.addJobprofile();
+    })
+
+    //visit inbox page and fetch latest emails
+    it('visit inbox page and fetch latest email', function () {
+        cy.server()
+        cy.route('GET',`http://localhost:8091/email/fetchByButton**`).as('fetch_emails')
+        cy.get("#fetchEmails").click();
+        cy.wait('@fetch_emails')
+    })
+
+    //check send email sucessfully fetched or not
+    it('check email is assigned to a tag or not', function () {
+        cy.get('#subnav .sub-title').contains(data.jobprofile);
+        cy.get('#subnav .subenav:first').find('a:first').should('not.have.text', 'all(0/0)');
+        cy.get('#subnav .subenav:first').find('a:first').click().then(function() {
+            cy.get('.emailstyle p').contains('ashutosh_m@excellencetechnologies.in')
```

Comment:
no hard coding

Reply:
this test case is not clear. how are u checking the automatic tags are working here?

in this test case you are only check if emails are present or not that all??

Reply:
sir first i checked that added tag should exist in inbox then clicked on that tag and then checked if sent email should exist there.

Reply:
in this test case i am checking, after clicking on tag email should exist in that tag.

Reply:
can you extend this test case to having 3 job profiles

Reply:
ok sir